packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,15 +1,65 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, hasFocus, hasInput, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary,
+        paddingTop: 0,
+        paddingBottom: 0,
+        selectors: {
+          ':hover ': {
+            borderColor: palette.neutralPrimary
```

Comment:
nit, but is it possible to snap to semantic color values here rather than palette colors? This would be a great forcing factor to ensure that TextField and SearchBox are identical. Could be a separate PR.

Reply:
Hello @dzearing ! That's a great point and I'll be making a PR for those changes shortly.